### DISCUSSION TITLE
MacOS ARM

### DISCUSSION THREAD

**Discussion by hrtl** [OrcaSlicer/OrcaSlicer#467] [General]
The whole  "xattr -d com.apple.quarantine" routine doesn't seem to work, nor does "Override Malware Protection" work for me. It looks like Apple has docked apps that are distributed via .zip archives. Is there a way of distributing .dmg, or .pkg files?
I have tried unzipping the archive on a different machine and copying to the M1 with the same result. Original Bambu Studio works fine.

Issue occurs on Mac Studio M1, Monterey 12.6.1

**Comment by umpmanjoe** [comment]
I'm having same issue own Ventura

**Comment by asucks** [comment]
Also having issues in Ventura 13.2.1 - MacBook Pro M2

**Comment by colorsone** [comment]
Here's a fix: https://github.com/SoftFever/BambuStudio-SoftFever/actions/runs/4234294979
